frontend/src/features/recommendation/components/spotItemList/SpotItemList.tsx

```
@@ -13,22 +13,27 @@ function SpotItemList({
   onSpotClick,
 }: SpotItemListProps) {
   return (
-    <div css={[flex({ direction: 'column', gap: 20 })]}>
+    <ul role="list" css={[flex({ direction: 'column', gap: 20 })]}>
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<ol>
<ul>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)